en/restart-a-tidb-cluster.md

```diff
@@ -85,13 +85,3 @@ To activate the graceful logoff feature, you need to enable some related configu
 2. Install or update TiDB Operator.
 
     To install or update TiDB Operator, see [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md).
-
```

Comment:
Please remove the whole section of https://docs.pingcap.com/tidb-in-kubernetes/dev/restart-a-tidb-cluster#gracefully-restart-a-single-pod-of-the-tidb-component